src/utilities/getNegativeNumbers.test.js

```diff
@@ -0,0 +1,24 @@
+import { getNegativeNumbers } from './getNegativeNumbers';
+describe('The getNegativeNumbers function', () => {
```

Comment:
There is a missing newline between `import` and `describe`. Did you use prettier on all of the files?